resources/user-agents/browsers/samsung/samsung-android-browser-6-2-on.json

```diff
@@ -22,14 +22,25 @@
         {
           "match": "Mozilla/5.0 (#PLATFORM##DEVICE# Build/*) applewebkit* (*khtml*like*gecko*) SamsungBrowser/#MAJORVER#.#MINORVER#*Chrome*Safari*",
           "devices": {
-            "SM-J320F*": "Samsung SM-J320F"
+            "SM-G920F*": "Samsung SM-G920F"
           },
           "platforms": [
-            "Android_7_1", "Android_7_0",
-            "Android_6_0",
-            "Android_5_1"
+            "Android_8_0",
+            "Android_7_0",
```

Comment:
This should have `Android_6_0` in the list no?

Reply:
Well, probably. I only added versions which listed on gsmarena, not adding anything else...

Reply:
Well, probably. I only added versions which listed on gsmarena, not adding anything else...

Reply:
I asked because you have it in the `chrome-android-43-on.json` file for this device.

FWIW, we do show that device on 6.0:

```
Mozilla/5.0 (Linux; Android 6.0.1; SM-G920F Build/MMB29K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/57.0.2987.132 Mobile Safari/537.36 
Mozilla/5.0 (Linux; Android 6.0.1; SAMSUNG SM-G920F Build/MMB29K) AppleWebKit/537.36 (KHTML, like Gecko) SamsungBrowser/5.4 Chrome/51.0.2704.106 Mobile Safari/537.36 
```

Reply:
Yeah, I didn't add it there at first but one testcase failed then :)